src/main/java/io/obswebsocket/community/client/OBSCommunicator.java

```diff
@@ -227,9 +228,17 @@ public void onHello(Hello hello) {
                 hello.isAuthenticationRequired()
         ));
 
+        // If RPC version doesn't match, then the protocol isn't supported
+        if(hello.getRpcVersion() != RPC_VERSION) {
```

Comment:
What we can already do here though is something like:
```java
if (hello.getRpcVersion() < RPC_VERSION) {
  // ERROR
}
```

Reply:
Agreed!